src/EditModel/option-set/OptionManagement.component.js

```diff
@@ -90,6 +97,11 @@ class OptionManagement extends Component {
         }
     }
 
+    handleFilter = ({ target: { value } }) => {
+        this.setState({ filter: value })
+        actions.getOptionsFor(this.props.model, value);
```

Comment:
Note that this is of course debounced. The easy debouncing of observables is something I will actually miss about this paradigm.